docs/reference/commandline/images.md

```diff
@@ -33,6 +33,7 @@ Options:
       --help            Print usage
       --no-trunc        Don't truncate output
   -q, --quiet           Only show numeric IDs
+      --sort-by string  Customize list order, sort by field[:order]. e.g. "ID:asc,CreatedAt:desc"
```

Comment:
I wonder if we should use always lowercase on the docs (and in the help), @thaJeztah wdyt ?